### DISCUSSION TITLE
A bunch Safari issues

### DISCUSSION THREAD

**Discussion by tluyben** [Budibase/budibase#3186] [Help]
I could not find any comprehensive overviews of all the issues, but on my m1, latest safari, there are quite a bunch of quirks for me with the latest released budibase. On the final site render (spacing between the menu items is gone) and in the builder: for me the builder does not load at all (maybe I need to wait even longer?) while on Chrome all is fine. No errors in the console. 

Is this known? Should I make an inventory?

**Comment by mjashanks** [comment]
Thanks @tluyben 

One of these is known: https://github.com/Budibase/budibase/issues/2945

Please go ahead and let us know of anything that you find :) screenshots would be helpful too.

**Comment by alexander-potemkin** [comment]
that looks like another one - just tried to create a template project from the get started page - that's how it looks for like 5 minutes now...

<img width="1562" alt="Screenshot 2021-11-04 at 18 36 08" src="https://user-images.githubusercontent.com/2323840/140360830-603ff5b2-ccf6-4163-b6ff-fe46070f35d5.png">

**Reply by mjashanks** [reply]
Yea - apologies, I think this is https://github.com/Budibase/budibase/issues/2945

Working on a fix. Works with all the other major browsers.

**Reply by alexander-potemkin** [reply]
yes, works on Brave, for example; thanks!
